### PR TITLE
Feat "error margin"

### PR DESCRIPTION
**Pull request by benib**
- allows to enter votes with error margin for either polls or extrapolated data
- A different visualisation is used in case `withErrorMargin` is true to show the uncertainty

### CODE REVIEW

File: views/HtmlStatic.html [hunk 1]
```diff
@@ -39,12 +41,37 @@
...
+        return maxErrorMarginValue;
+      },
       titleClassAttribute: ({ toolRuntimeConfig }) => {
         if (toolRuntimeConfig.displayOptions && toolRuntimeConfig.displayOptions.hideTitle) {
           return 'q-election-executive-title--hidden';
         }
         return '';
       },
+      //TODO: do not random sort but return by array index if no values given
       sortedCandidates: ({ candidates }) => {
```
Comment (manuelroth) [inline views/HtmlStatic.html]: Candidates should if available additionally be sorted on the errorMargin property. See -> 

Bildschirmfoto 2019-03-20 um 17 23 40

Comment (benib) [reply views/HtmlStatic.html]: Thanks for spotting. This is now fixed.

File: resources/locales/en/translation.json [hunk 2]
```diff
@@ -8,12 +8,18 @@
   "Nur eintragen, sofern für den Ausgang der Wahl relevant&#46;": "only enter if relevant to the outcome of the election&#46;",
   "Stimmen liegen in % anstatt absoluten Zahlen vor": "votes are in % instead of absolute numbers",
   "Verfügbare Sitze": "available seats",
+  "Daten mit Fehlermarge": "Data with error margin",
```
Comment (manuelroth) [inline resources/locales/en/translation.json]: I think we should consistently use "Fehlerbereich" or "Fehlermarge" and not use both. In q-election-executive we just say "mit Fehlerbereich" and not "Daten mit Fehlermarge". I think this should also be consistent between the tools

Comment (benib) [reply resources/locales/en/translation.json]: done

**Pull request merged**